CMakeLists.txt

```diff
@@ -139,9 +139,17 @@ add_custom_command(
 
 add_custom_target(phony_always_runs ALL DEPENDS ${CMAKE_BINARY_DIR}/${_script_file})
 
-# todo:: make this generated
+
+# AUTO-GENERATE lanterndb.control file for PostgreSQL
+set(CONTROL_TEMPLATE "${CMAKE_SOURCE_DIR}/cmake/lanterndb.control.template")
+set(CONTROL_OUTPUT "${CMAKE_SOURCE_DIR}/lanterndb.control")
+
+configure_file(${CONTROL_TEMPLATE} ${CONTROL_OUTPUT})
+install(FILES ${CONTROL_OUTPUT} DESTINATION ${PostgreSQL_EXTENSION_DIR})
+
 set(_control_file "lanterndb.control")
```

Comment:
This will need updating accordingly, once `lanterndb.control` moves to the build directory

Reply:
Fixed